packages/react-composites/src/composites/CallComposite/components/CallPane.tsx

```diff
@@ -0,0 +1,195 @@
+      <PeoplePaneContent
+        {...props}
+        onRemoveParticipant={removeParticipantFromCall}
+        setDrawerMenuItems={setDrawerMenuItems}
+        strings={strings}
+      />
+    </CallAdapterProvider>
+  );
+
+  // Use document.getElementById until Fluent's Stack supports componentRef property: https://github.com/microsoft/fluentui/issues/20410
+  const modalLayerHostElement = document.getElementById(props.modalLayerHostId);
+  const modalHostRef = useRef<HTMLElement>(modalLayerHostElement);
+  const modalHostWidth = _useContainerWidth(modalHostRef);
+  const modalHostHeight = _useContainerHeight(modalHostRef);
+  const minDragPosition: _ICoordinates | undefined = useMemo(
+    () =>
+      modalHostWidth === undefined
+        ? undefined
+        : {
+            x: props.rtl ? -1 * modalPipRightPositionPx : modalPipRightPositionPx - modalHostWidth + modalPipWidthPx,
+            y: -1 * modalPipTopPositionPx
+          },
+    [modalHostWidth, props.rtl]
+  );
+  const maxDragPosition: _ICoordinates | undefined = useMemo(
+    () =>
+      modalHostWidth === undefined || modalHostHeight === undefined
+        ? undefined
+        : {
+            x: props.rtl ? modalHostWidth - modalPipRightPositionPx - modalPipWidthPx : modalPipRightPositionPx,
+            y: modalHostHeight - modalPipTopPositionPx - modalPipHeightPx
+          },
+    [modalHostHeight, modalHostWidth, props.rtl]
+  );
+
+  const pipStyles = useMemo(() => getPipStyles(theme), [theme]);
+
+  const dataUiId = props.activePane === 'people' ? 'call-composite-people-pane' : '';
+
+  return (
+    <Stack verticalFill grow styles={paneStyles} data-ui-id={dataUiId} tokens={props.mobileView ? {} : sidePaneTokens}>
+      {header}
+      <Stack.Item verticalFill grow styles={paneBodyContainer}>
+        <Stack horizontal styles={scrollableContainer}>
+          <Stack.Item verticalFill styles={scrollableContainerContents}>
+            <Stack styles={props.activePane === 'people' ? availableSpaceStyles : hiddenStyles}>{peopleContent}</Stack>
+          </Stack.Item>
+        </Stack>
+      </Stack.Item>
+      {props.mobileView && (
+        <ModalLocalAndRemotePIP
+          callAdapter={props.callAdapter}
+          modalLayerHostId={props.modalLayerHostId}
+          hidden={hidden}
+          styles={pipStyles}
+          minDragPosition={minDragPosition}
+          maxDragPosition={maxDragPosition}
+        />
+      )}
+      {drawerMenuItems.length > 0 && (
+        <Stack styles={drawerContainerStyles}>
+          <_DrawerMenu onLightDismiss={() => setDrawerMenuItems([])} items={drawerMenuItems} />
+        </Stack>
+      )}
+    </Stack>
+  );
+};
+
+/**
+ * Active tab option type for {@link CallPane} component
+ * @private
+ */
+/** @beta */
+export type CallPaneOption = 'none' | 'people';
+
+const hiddenStyles: IStackStyles = {
+  root: {
+    display: 'none'
+  }
+};
+
+const sidePaneStyles: IStackStyles = {
```

Comment:
Possible to re-use these styles by moving them to `common/styles`?

Reply:
Moved to common/styles/Pane.styles.ts